modules/packages/URL.chpl

```diff
@@ -133,26 +150,23 @@ module URL {
                 corresponding parameter of the :record:`~IO.fileWriter` type.
                 Defaults to true, but when safe, setting it to false
                 can improve performance.
-  :arg start: zero-based byte offset indicating where in the file the
-              fileWriter should start writing. Defaults to 0.
-  :arg end: zero-based byte offset indicating where in the file the
-            fileWriter should no longer be allowed to write. Defaults
-            to a ``max(int)`` - meaning no end point.
+  :arg region: zero-based byte offset indicating where in the file the
+               fileWriter should start and stop writing. Defaults to
+               ``0..`` - meaning from the start of the file to no end point.
   :returns: an open fileWriter to the requested resource.
 
   :throws SystemError: Thrown if a fileWriter could not be returned.
   */
   proc openUrlWriter(url:string, param locking=true,
-                 start:int(64) = 0, end:int(64) = max(int(64)))
+                 region: range(?) = 0..)
                 : fileWriter(locking) throws {
-    return openUrlWriterHelper(url, _iokind.dynamic, locking, start, end);
+    return openUrlWriterHelper(url, _iokind.dynamic, locking, region);
   }
 
   private proc openUrlWriterHelper(url:string,
                                    param kind=_iokind.dynamic,
                                    param locking=true,
-                                   start:int(64) = 0,
-                                   end:int(64) = max(int(64)),
+                                   region: range(?) = 0..,
                                    style:iostyleInternal = defaultIOStyleInternal())
     : fileWriter(kind, locking) throws {
 
```

Comment:
And ditto here about the `start` and `end` arguments still being referenced in the body of the function

Reply:
Same as above